playbooks/resize_syspart.yml

```diff
@@ -54,7 +54,7 @@
 #
 #####################################################################################
 - name: Resize System Partition (Windows)
-  hosts: windows_box
+  hosts: windows_box:&vms
```

Comment:
there is a group named vm_wrk_win which wich will yield the same result as windows_box:&vms. I know conceptually these are two <> objects. Just saying here.

Reply:
Done.